pkgs/racket-doc/file/scribblings/glob.scrbl

```diff
@@ -117,6 +119,19 @@ Examples:
   This operation accesses the filesystem.
 }
 
+@defproc*[([(glob-quote [pattern string?]) string?]
+           [(glob-quote [pattern path?]) path?]
+)]{
+  Escapes all @tech{glob wildcards} in the given string or path string,
+  but not braces.
```

Comment:
How would you make a glob that should match only files that have open-curly-braces in them? If it's `"\\{"`, does `(glob-quote "{")` produce that?

Reply:
`(glob "{" )` should work

but probably, `glob-quote` should optionally escape `{` `,` and `}` characters too